### ISSUE TITLE
InvalidConfiguration error Value "True" is not valid for CUBEJS_SCHEDULED_REFRESH_TIMER

### ISSUE THREAD

**Issue by craigbeck** [cube-js/cube.js#1928]
**Describe the bug**

1. Updated to latest CubeJs dependencies `@cubejs-backend/server@0.25.30` 
2. built docker image (not using "official" docker image for CubeJs yet)
3. deployed to Google AppEngine Flex environment via CI build process
4. Container fails to start with error:
  ```
    InvalidConfiguration [Error]: Value "True" is not valid for CUBEJS_SCHEDULED_REFRESH_TIMER. Should be boolean or number (in seconds) or string in time format (1s, 1m, 1h)
  574    at convertTimeStrToMs (/node_modules/@cubejs-backend/shared/dist/src/env.js:26:11)
  575    at asBoolOrTime (/node_modules/@cubejs-backend/shared/dist/src/env.js:53:12)
  576    at Object.refreshTimer (/node_modules/@cubejs-backend/shared/dist/src/env.js:68:12)
  577    at Object.getEnv (/node_modules/@cubejs-backend/shared/dist/src/env.js:91:26)
  578    at new CubejsServerCore (/node_modules/@cubejs-backend/server-core/dist/src/core/server.js:130:114)
  579    at Function.create (/node_modules/@cubejs-backend/server-core/dist/src/core/server.js:259:16)
  580    at new CubejsServer (/node_modules/@cubejs-backend/server/dist/server.js:32:43)
  581    at Object.<anonymous> (/index.js:50:16)
  582    at Module._compile (internal/modules/cjs/loader.js:1063:30)
  583    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
  ```

Env var for `CUBEJS_SCHEDULED_REFRESH_TIMER` set via `app.yaml` file and remained unchanged from previous working value:
  ```
  CUBEJS_SCHEDULED_REFRESH_TIMER: true
  ```
Value is lowercase unlike in error message.

**To Reproduce**
see above

**Expected behavior**
Expected CubeJs to start normally with `CUBEJS_SCHEDULED_REFRESH_TIMER=true`

**Screenshots**
n/a

**Version:**
@cubejs-backend/server@0.25.30

**Additional context**

**Comment by craigbeck** [comment]
as reported via CubeJs Slack https://cube-js.slack.com/archives/CLGEK5N1K/p1611789847002300

**Comment by ovr** [comment]
Hello @craigbeck ,

I saw that message in slack, and I've prepared a fix for that - https://github.com/cube-js/cube.js/commit/99a5759e619824666b48c589a5c98c82c1817025

But I forget to reply in that thread. Fixed in `0.25.32`.

Thanks

**Comment by craigbeck** [comment]
Thanks!
